apps/marketplace/components/Brief/BriefSpecialistResponseForm2.js

```diff
@@ -48,8 +46,8 @@ const BriefSpecialistResponseForm2 = ({
     <DocumentTitle title="Brief Response - Digital Marketplace">
       <div className="col-sm-push-2 col-sm-8 col-xs-12">
         <article role="main">
-          {(briefResponseSuccess && !addAnotherSpecialist) || briefResponses.length >= MaxSpecialists ? (
-            <Redirect to={`${match.url}/specialist/respond/submitted`} />
+          {(briefResponseSuccess && !addAnotherSpecialist) || briefResponses.length >= brief.numberOfSuppliers ? (
+            <Redirect to={`${match.url}/specialist2/respond/submitted`} />
```

Comment:
<Redirect to={`${match.url}/**_specialist2_**/respond/submitted`} />  change to  <Redirect to={`${match.url}/**_specialist_**/respond/submitted`} />

There seems no need for 2 to be in the html link